### PR TITLE
Inverse kinematics validation check

### PR DESCRIPTION
**Pull request by Jakub-Ivan-STUBA**
Fix for IK solver, we now validate the collision for the computed joint values. If robot does not collide, we check the distance betweenn the desired pose and the computed solution for the end effector. If the computed solution satisfies requirements, we return success = true in response of the service call ik/get_solution.

Added recomputing possibility in order to attempt to calculate the ik solution if the previous tries failed. The number of the attempts will be defined in the header for the IK solver class, recommended value is 3. We have to make sure that if not success in calculating the IK in all the attempts, the time of service call will be the sum of all tries therefore be careful when editing the preset timeout for the IK solver.

### CODE REVIEW

File: ROS_ws/src/sprinter_description/launch/display_sprinter.launch
Comment (patrik-knaperek) [inline ROS_ws/src/sprinter_description/launch/display_sprinter.launch:1]: This file is no longer needed as it was renamed to display.launch

File: ROS_ws/src/printer/printer_control/launch/printer_control.launch
Comment (patrik-knaperek) [inline ROS_ws/src/printer/printer_control/launch/printer_control.launch:1]: printer_control.launch should launch only the printer_control and printer_ik_solver nodes

Comment (Jakub-Ivan-STUBA) [reply ROS_ws/src/printer/printer_control/launch/printer_control.launch:1]: dropped commit should resolve this

File: ROS_ws/src/sprinter_description/launch/display_sprinter.launch
Comment (Jakub-Ivan-STUBA) [reply ROS_ws/src/sprinter_description/launch/display_sprinter.launch:1]: dropped commit should resolve this

**Pull request closed without merging**